components/music/AlbumView.bs

```diff
@@ -106,18 +107,18 @@ sub setOnScreenTextValues(json)
             adjustScreenForNoOverview()
         end if
 
-        setFieldTextValue("numberofsongs", stri(json.ChildCount) + " Tracks")
+        setFieldTextValue("numberofsongs", tr("%1 Tracks").Replace("%1", stri(json.ChildCount)))
```

Comment:
```suggestion
        setFieldTextValue("numberofsongs", `${json.ChildCount} ${tr("Tracks")}`)
```

Additionally, "Tracks" must be added to the en_us translation file.

Use this same pattern for other items below.